app/views/play/CampaignView.coffee

```diff
@@ -120,7 +120,7 @@ module.exports = class CampaignView extends RootView
               noty({text: 'Please create or join a classroom first', layout: 'topCenter', timeout: 8000, type: 'success'})
           return
       if @terrain is 'game-dev-hoc'
-        window.tracker?.trackEvent 'Started Campaign', category: 'Amazon', label: 'game-dev-hoc'
+        window.tracker?.trackEvent 'Start HoC Campaign', label: 'game-dev-hoc'
```

Comment:
This still only works for `game-dev-hoc`. Remove the if check here and have `label` property be `@terrain`. I think then this event will fire for each HoC activity, not just the AWS one.